source/renderer/app/components/wallet/WalletRestoreDialog.js

```diff
@@ -551,6 +628,21 @@ export default class WalletRestoreDialog extends Component<Props, State> {
     return this.state.walletType === WALLET_RESTORE_TYPES.LEGACY;
   }
 
+  isYoroiBalance() {
```

Comment:
@DeeJayElly please rename to `isYoroiLegacy()`

Reply:
Done